jdaviz/core/loaders/parsers/specutils.py

```diff
@@ -0,0 +1,40 @@
+from functools import cached_property
+from specutils import Spectrum1D, SpectrumList
+
+from jdaviz.core.loaders.parsers import BaseParser
+from jdaviz.core.registries import loader_parser_registry
+
+
+@loader_parser_registry('specutils.Spectrum')
+class SpecutilsSpectrumParser(BaseParser):
+    SpecutilsCls = Spectrum1D
+
+    @property
+    def is_valid(self):
+        if self.app.config not in ('specviz', 'specviz2d'):
+            # NOTE: temporary during deconfig process
+            return False
+        try:
+            self.output
+        except Exception as e:
+            print(f"{self.SpecutilsCls.__name__} read failed", str(e))
```

Comment:
If you must keep this message, maybe send to snackbar or something?
```suggestion
```

Reply:
just a missed debugging artefact, will remove